nemo_text_processing/text_normalization/zh/verbalizers/time.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pynini
+from nemo_text_processing.text_normalization.zh.graph_utils import (
+    NEMO_DIGIT,
+    NEMO_NOT_QUOTE,
+    NEMO_SIGMA,
+    GraphFst,
+    delete_space,
+    insert_space,
+)
+from nemo_text_processing.text_normalization.zh.utils import UNIT_1e01, get_abs_path
+from pynini.lib import pynutil
+
+
+class Time(GraphFst):
+    '''
+        tokens { time { h: "1" m: "02" s: "36" } } -> 一点零二分三十六秒
+        tokens { time { suffix "am"  hours: "1" minutes: "02" seconds: "36" } } -> 上午一点零二分三十六秒
+    '''
+
+    def __init__(self, deterministic: bool = True, lm: bool = False):
+        super().__init__(name="time", kind="verbalize", deterministic=deterministic)
+        graph_digit = pynini.string_file(get_abs_path("data/cardinal/digit.tsv"))
+        graph_ten = pynini.string_file(get_abs_path("data/cardinal/digit_teen.tsv"))
```

Comment:
I observed there are three places loaded `"data/cardinal/digit_teen.tsv"`, but they declared two different variables below, please unify the name space. 
* `graph_ten`: line 36 in `text_normalization/zh/verbalizers/time.py`; line 30 in `text_normalization/zh/verbalizers/date.py`.
* `graph_teen`: line 48 in `text_normalization/zh/taggers/cardinal.py`.